File: public/extras/generateMarks.js
function generateMarks(email,semester){
    //Generate random semester scores and return JSON object
    let marks = []
    for(let i=1;i<semester;i++){
        var sem_marks = {}
        let random = Math.floor(Math.random() * (3))+5  + 0.1*Math.floor(Math.random() * (9))+1
        sem_marks[i] = random
        marks.push(sem_marks)
    }
    return {
        email: email,
        semester: marks
    }
}

console.log(generateMarks('vihardevalla@gmail.com',8))

File: public/extras/generateMarks.py
from random import randint
# Generate random marks for a student
def generate(email,sem):
    marks = []
    for i in range(1,sem):
        cpga = randint(6,9)+randint(0,9)*0.1
        marks.append({i:cpga})
    dic = {"email":email,"semester":marks}
    return dic
print(generate('gsstrivikram18@gmail.com',5))